\chapter{Discussion and Conclusion}
\label{chap:conclusion}

As infectious diseases continue to be of growing interest and concern \citep{Baker2022-eb}, methods to monitor, diagnose, understand, and combat them must continue to evolve and improve. Sequencing technologies have developed at breakneck pace in recent decades \citep{Hu2021-dd, Schatz2013-vw}, and sequencing based investigative methods have been applied to great effect in almost all fields of biology and medicine. I have taken advantage of the unique properties of the most recent generation of sequencing technology for infectious disease applications, using it to identify and surveille AMR genes, assemble eukaryotic pathogen genomes, and link plasmids to hosts in complex microbial communities.

While nanopore sequencing can detect specific AMR genes quickly and agnostically \citep{Tamma2019-jg}, it can still be difficult to infer phenotypic resistance \citep{Yee2021-td}. Genomic data is able to provide information on an organismâ€™s potential behavior, but observations of actual activity and function require transcriptomic or proteomic data. In situ functional studies of resistance mechanisms with metatranscriptomics could help to address these issues. As understanding of resistance mechanisms continues to grow, predictions of phenotypic resistance will become increasingly accurate and clinically actionable.

Use of long read sequencing for genome assembly has unlocked continuously larger genomes \citep{Neale2014-di}, and accompanying software has made high quality genome assembly more accessible than ever before \citep{Fan2021-cq}. As more and more eukaryotic pathogen genomes are sequenced, collated, and curated \citep{Aurrecoechea2017-tl}, they will become crucial to the development of sequencing-based diagnostics of infectious diseases \citep{Lu2018-wr}, in addition to being useful for furthering basic science research on these organisms.

Long-read data for metagenomic assembly not only produces longer contigs, but is able to preserve base modification data, which can be used for binning applications. Not only can contigs be grouped on the bases of base modifications, but reads can as well. Although this has not been shown using a single nanopore run, its possibility has been demonstrated using PacBio data \citep{Beaulaurier2018-mu}. Developing this capability in nanopore sequencing would enable multi-host plasmid assignments which are currently unfeasible.
